ChangeLog

```diff
@@ -1,3 +1,11 @@
+2021-09-08  Dirk Eddelbuettel  <edd@debian.org>
+
+	* DESCRIPTION (Version, Date): Roll minor version
+
+	* inst/include/conditionalSampler.h: Use std::pow
+
+	* README.md: Belatedly add Ivan
```

Comment:
__*Ilya*__ not Ivan (usually I don't mind at all, happened often, but it's the project's ChangeLog :) )

Reply:
Yeebus.  I am so sorry.  I was just fixing / preparing slides where I praised (for another project) another (former) GSoC student and was indeed named Ivan.  I apologise and will fix.

Reply:
Fixed and force-pushed.  At least I got it right in the README :)